source/data/build.R

```diff
@@ -1,7 +1,10 @@
-args = commandArgs(TRUE)
+library(yaml)
+PATHS <- yaml.load_file('constants.yaml')
+args  <- commandArgs(TRUE)
+
 main <- function(i){
     x    <- seq(-10, 10, 0.1)
-    write.table(x, sprintf("build/data/%s.txt", i), sep = "|", 
+    write.table(x, sprintf("%s/%s.txt", PATHS$build$data, i), sep = "|", 
```

Comment:
We seem to be having some trouble with floating point precision for numbers less than one with this command. Probably not what we want to be demonstrating.